charts/aergia/templates/deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "aergia.labels" . | nindent 4 }}
 spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
If you're doing this, can you add to the `values.yaml` file the default of ```replicaCount: 1``` so that if it isn't defined in a users custom values, it will always use 1.